understanding/22/pointer-target-spacing.html

```diff
@@ -0,0 +1,153 @@
+            <figcaption><strong>Fail:</strong> Two rows of buttons, all with a height of only 20 CSS pixels. In the top row, buttons all have a width of more than 44 CSS pixels. For the top row, targets above and below are in close proximity. The top spacing of the first row is 4 px while the bottom spacing is 2 px, which adds up to 26 px for target height and vertical spacing combined. Therefore, the buttons in the top row fail the Success Criterion. The button "Pop" in the second row is only 32px wide with no horizontal spacing, so it also fails the Success Criterion even when there is sufficient space below to meet the vertical spacing requirement.</figcaption>
+        </figure>
+
+        <p>Where targets are embedded inside other targets, the inner target either needs a size of at least 44 x 44 CSS pixels or must placed at the edge or corner in a way that sufficient adjacent spacing is available in each dimension.</p>
+
+        <figure id="pointer=target-embedded">
+            <img src="img/pointer-target-example5.png" alt="Three examples of a target adjacent to another target (see legend below)" />
+            <figcaption>Three examples of a person's profile image, which is supposed to be a target leading to another page, and an magnifying glass icon, which is supposed to open a lightbox view of the profile image. In the first example, the magnifying icon is outside the picture and passes. In the second example, the icon overlaps with the top-left corner of the image, but passes due to sufficient spacing top and left. In the third example, the icon sits inside the linked image. Since it is surrounded by an active click area, the icon target is not large enough to pass the success criterion.</figcaption>
+        </figure>
+      
+        <p>Users with different disabilities have different needs for control sizes. It can be beneficial to provide an option to increase the active target area without increasing the visible target size. Another option is to provide a mechanism to control the density of layout and thereby change target size or spacing, or both. This can be beneficial for a wide range of users. For example, users with visual field loss may prefer a more condensed layout with smaller sized controls while users with other forms of low vision may prefer large controls.</p>
+      
+      <p>Where user-agents control the size of a target, such as the options in a <code>select</code> drop-down, and the author has not tried to change the size of target with size or padding adjustments, the target is excempted from this criterion. It would be beneficial for users if the minumum size &amp; spacing requirement is meet, but where the user-agent has defined the target it is not the authors responsibility.</p>
+        
+        
+    </section>
+    <section id="benefits">
+      <h2>Benefits of Pointer Target Spacing</h2>
+      
+      <ul>
```

Comment:
could do with a single line of text before the list...as this currently doesn't list "Benefits", but "Who benefits".

something like "Having targets with sufficient target spacing can help all users who may have difficulty in confidently targeting/operating small controls. These include, but aren't limited to:"